tests/std/tests/P1004R2_constexpr_vector/test.cpp

```diff
@@ -0,0 +1,703 @@
+
+        const auto cat = const_range_constructed.at(2);
+        static_assert(is_same_v<remove_const_t<decltype(cat)>, int>);
+        assert(cat == 2);
+
+        const auto op = range_constructed[3];
+        static_assert(is_same_v<remove_const_t<decltype(op)>, int>);
+        assert(op == 3);
+
+        range_constructed[3] = 4;
+        const auto op2       = range_constructed[3];
+        static_assert(is_same_v<remove_const_t<decltype(op2)>, int>);
+        assert(op2 == 4);
+
+        const auto cop = const_range_constructed[3];
+        static_assert(is_same_v<remove_const_t<decltype(cop)>, int>);
+        assert(cop == 3);
+
+        const auto f = range_constructed.front();
+        static_assert(is_same_v<remove_const_t<decltype(f)>, int>);
+        assert(f == 0);
+
+        const auto cf = const_range_constructed.front();
+        static_assert(is_same_v<remove_const_t<decltype(cf)>, int>);
+        assert(cf == 0);
+
+        const auto b = range_constructed.back();
+        static_assert(is_same_v<remove_const_t<decltype(b)>, int>);
+        assert(b == 5);
+
+        const auto cb = const_range_constructed.back();
+        static_assert(is_same_v<remove_const_t<decltype(cb)>, int>);
+        assert(cb == 5);
+
+        const auto d = range_constructed.data();
+        static_assert(is_same_v<remove_const_t<decltype(d)>, int*>);
+        assert(*d == 0);
+
+        const auto cd = const_range_constructed.data();
+        static_assert(is_same_v<remove_const_t<decltype(cd)>, const int*>);
+        assert(*cd == 0);
+    }
+
+    { // capacity
+        vec range_constructed(begin(input), end(input));
+
+        const auto e = range_constructed.empty();
+        static_assert(is_same_v<remove_const_t<decltype(e)>, bool>);
+        assert(!e);
+
+        const auto s = range_constructed.size();
+        static_assert(is_same_v<remove_const_t<decltype(s)>, std::size_t>);
+        assert(s == size(input));
+
+        const auto ms = range_constructed.max_size();
+        static_assert(is_same_v<remove_const_t<decltype(ms)>, std::size_t>);
+        assert(ms == static_cast<size_t>(-1) / 4);
+
+        range_constructed.reserve(20);
+
+        const auto c = range_constructed.capacity();
+        static_assert(is_same_v<remove_const_t<decltype(c)>, std::size_t>);
+        assert(c == 20);
+
+        range_constructed.shrink_to_fit();
+
+        const auto c2 = range_constructed.capacity();
+        static_assert(is_same_v<remove_const_t<decltype(c2)>, std::size_t>);
+        assert(c2 == 6);
+    }
+
+    { // modifiers
+        vec range_constructed(begin(input), end(input));
+
+        vec cleared = range_constructed;
+        cleared.clear();
+        assert(cleared.empty());
+        assert(cleared.capacity() == range_constructed.capacity());
+
+        vec inserted;
+
+#ifndef __EDG__ // TRANSITION, VSO-1273386, VSO-1274387
+        const int to_be_inserted = 3;
+        inserted.insert(inserted.begin(), to_be_inserted);

```

Comment:
Are we covering `iterator insert(const_iterator position, size_type n, const T& x)` in testing anywhere?

Reply:
Thanks, it indeed seems to be missing. I see that you already added it ;)